src/js/edu-benefits/1990-rjsf/config/form.js

```diff
@@ -24,20 +27,31 @@ const formConfig = {
   title: 'Apply for education benefits',
   subTitle: 'Form 22-1990',
   chapters: {
-    veteranInformation: {
-      title: 'Veteran Information',
+    applicantInformation: {
+      title: 'Applicant Information',
       pages: {
-        veteranInformation: {
-          title: 'Veteran information',
-          path: 'veteran-information',
+        applicantInformation: _.merge(applicantInformation(fullSchema1990, {
+          isVeteran: true,
+          fields: [
+            'veteranFullName',
+            'veteranSocialSecurityNumber',
+            'veteranDateOfBirth',
+            'gender'
+          ],
+        }), {
           uiSchema: {
-          },
-          schema: {
-            type: 'object',
-            properties: {
+            veteranDateOfBirth: {
+              'ui:validations': [
+                (errors, dob) => {
+                  // TODO: Use a constant instead of 'YYYY-MM-DD'
+                  if (moment(dob, 'YYYY-MM-DD').isAfter(moment().endOf('day').subtract(17, 'years'))) {
```

Comment:
I'm not too worried about the format being incorrect, just if it's undefined or not. All the dates come from our date widget or backend prefill code, so I'm pretty confident that we're always going to have an ISO date, if it's not empty.

So I was at most thinking of a check that the date is undefined, or that it matches `/\d{4}-\d{2}-\d{2}/`, just so we're explicitly handling the empty case.

But like I said, I'm not sure that's worth it.